### PR TITLE
Add back the "tag name" element location strategy.

### PR DESCRIPTION
**Pull request by shs96c**
Closes issue #419.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/490)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -4098,6 +4103,23 @@
...
 
 <section>
+<h4>Tag Name</h4>
+
+<p>To find a <a>web element</a> with the <dfn>Tag
+ Name</dfn> <a>strategy</a> the following steps need to be completed:
+
+<ol>
+ <li><p><p>Let <var>elements</var> be the result of
+ calling <a>getElementsByTagName</a> with
```
Comment (andreastt) [inline webdriver-spec.html]: Missing reference to https://dom.spec.whatwg.org/#dom-element-getelementsbytagname.

File: webdriver-spec.html [hunk 2]
```diff
@@ -4098,6 +4103,23 @@
...
+ argument <var>selector</var>, with the <a>context object</a> equal to
+ the <var>start node</var>.
+
+ <li><p>Return <var>elements</var>.
```
Comment (andreastt) [inline webdriver-spec.html]: I don’t think we need the steps here at all: all this does is assign to _elements_, then returning it.

The above can be shortened to:

> Fo find an element with the Tag Name strategy, return the result of calling _start node_’s _getElementsByTagName_ with the argument _selector_.

File: webdriver-spec.html (hunk 1 above)
Comment (shs96c) [inline webdriver-spec.html]: Fixed.

File: webdriver-spec.html (hunk 2 above)
Comment (shs96c) [inline webdriver-spec.html]: Done

**Comment by shs96c** [comment]
Apparently, rebasing and force pushing causes GH to not realise that reviews have been acted upon.

File: webdriver-spec.html [hunk 3]
```diff
@@ -128,6 +128,7 @@
    <!-- Fire an event --> <li><dfn data-lt="fire|fires|fired|event fires|event fired"><a href=https://dom.spec.whatwg.org/#concept-event-fire>Fire an event</a></dfn>
    <!-- Get an attribute by name --> <li><dfn data-lt="getting an attribute by name"><a href=https://dom.spec.whatwg.org/#concept-element-attributes-get-by-name>Get an attribute by name</a></dfn>
    <!-- getAttribute --> <li><dfn><a href=https://dom.spec.whatwg.org/#dom-element-getattribute>getAttribute</a></dfn>
+   <!-- getElementsByTagName --> <li><dfn><a href="https://dom.spec.whatwg.org/#dom-element-getelementsbytagname">getElementsByTagName</a></dfn>
```
Comment (AutomatedTester) [inline webdriver-spec.html]: can you wrap `getElementsByTagName` in `<code>` tag e.g. `<code>getElementsByTagName</code>` and then update the text below to wrap it there too.

Comment (shs96c) [inline webdriver-spec.html]: Done

**Pull request merged**